### ISSUE TITLE
Shears look whacky when held

### ISSUE THREAD

**Issue by FewerFlaws** [tr7zw/FirstPersonModel#66]
Shears need a bit of special treatment so that they're held in a way that makes sense.

![2021-01-06_15 21 50](https://user-images.githubusercontent.com/75643699/103832433-1d4be780-5033-11eb-827b-bcfd9105c441.png)
![2021-01-06_15 21 39](https://user-images.githubusercontent.com/75643699/103832436-1de47e00-5033-11eb-80c5-5cc32f11a943.png)

**Comment by tr7zw** [comment]
Noooot sure there is much I can do about that right now. I guess turning it sideways would look better? https://github.com/tr7zw/NotEnoughAnimations

**Comment by FewerFlaws** [comment]
Yep, turning them so that they're vertical rather than horizontal would be much better =]

**Comment by tr7zw** [comment]
Update: I think it would be better to solve that with an resource pack, rather than hardcoded flip the model. That should be the same way swords etc are done.
